### PR TITLE
Created address.py and used bs4

### PR DESCRIPTION
**Pull request by prdp1137**
Updated address.txt and made new address.txt and used dictionary to manage.

Also, bs4 is used for fetching different values.

Can you guys have a look?

### CODE REVIEW

File: NEA.py [hunk 1]
```diff
@@ -10,100 +20,96 @@
...
+except Exception:
+    print("NEA Location not found!")
+    exit(0)
 
 def exit2():
 
-	print('\n\033[94m [+] Our Site: www.askbuddie.com\n [+] Join Us: www.fb.com/groups/askbuddie\n [+] Our FB page: www.fbcom/askbuddie\n [+] ABOS: https://github.com/askbuddie')
-	print('\n\033[93m Thanks for using NEA Bill Checker...\n\n')
-	exit()
+    print('\n [+] Our Site: www.askbuddie.com\n [+] Join Us: www.fb.com/groups/askbuddie\n [+] Our FB page: www.fbcom/askbuddie\n [+] ABOS: https://github.com/askbuddie')
```
Comment (hemantapokharel) [inline NEA.py]:
```suggestion
    print('\n [+] Our Site: www.askbuddie.com\n [+] Join Us: www.fb.com/groups/askbuddie\n [+] Our FB page: www.fb.com/askbuddie\n [+] ABOS: https://github.com/askbuddie')
```

File: NEA.py [hunk 2]
```diff
@@ -10,100 +20,96 @@
...
+    r = requests.post(
+        'https://www.neabilling.com/viewonline/viewonlineresult/', params=parameters)
+except Exception:
+    print("\n\033[91m [-] Something went Wrong, Try again later.")
+    exit2()
 
-print  ' _____________________________________' 
+if 'No Records' in r.text:
+    print(
+        "\n\033[91m [-] Record not found. You entered\n wrong information.")
```
Comment (hemantapokharel) [inline NEA.py]:
```suggestion
        "\n[-] Record not found. You entered\n wrong information.")
```

File: NEA.py [hunk 3]
```diff
@@ -10,100 +20,96 @@
...
-	else:
-		print add + " Not found!"
-		exit(1)
+parameters = {'NEA_location': ad, 'sc_no': sc,
+              'consumer_id': custid, 'fromdate': '2/28/2017', 'todate': date}
+try:
+    r = requests.post(
+        'https://www.neabilling.com/viewonline/viewonlineresult/', params=parameters)
+except Exception:
+    print("\n\033[91m [-] Something went Wrong, Try again later.")
```
Comment (hemantapokharel) [inline NEA.py]:
```suggestion
    print("\n[-] Something went Wrong, Try again later.")
```

File: NEA.py [hunk 4]
```diff
@@ -10,100 +20,96 @@
...
 
-with open(file) as a:
-	x = a.read()
-	address = ''
-	address1 = add
-	address2 = add + " DC"
+try:
+    int(custid)
+except Exception:
+    print("\n\033[91m [-] Customer id must be interger.")
```
Comment (hemantapokharel) [inline NEA.py]:
```suggestion
    print("\n[-] Customer id must be interger.")
```

File: NEA.py [hunk 5]
```diff
@@ -10,100 +20,96 @@
...
-add = raw_input("\033[92m Enter your NEA Location : \033[94m").upper()
+add = input("Enter your NEA Location : ").upper()
 
-file = 'address.txt'
+ad = getAreaCode(add)
+
+try:
+    int(ad)
+except Exception:
+    print("NEA Location not found!")
```
Comment (hemantapokharel) [inline NEA.py]:
```suggestion
    print("[-] NEA Location not found!")
```

File: NEA.py [hunk 6]
```diff
@@ -10,100 +20,96 @@
...
+    exit(0)
```
Comment (hemantapokharel) [inline NEA.py]:
```suggestion
    exit2()
```

**Review by hemantapokharel — COMMENTED** [review]
This is looking great and simple now!!🌟

**Comment by TheBinitGhimire** [comment]
Hello @pr0d33p,

Everything seems fine as of now. I am looking forward to seeing this PR getting merged.

Thanks,
**Binit**

**Review by TheBinitGhimire — APPROVED** [review]
Hello @pr0d33p,

Everything seems fine as of now. I am looking forward to seeing this PR getting merged.

Thanks,
**Binit**

**Comment by prdp1137** [comment]
Hi, 

Now I'm merging to master. Thanks for your reviews guys @hemantapokharel @TheBinitGhimire.

**Pull request merged**